podman/podman-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: '3.8'
+
+# Intended to be run from the podman directory
+
+services:
+
+  kafka:
+      image: quay.io/strimzi/kafka:latest-kafka-3.7.0
+      ports:
+        - "9092:9092"
+      volumes:
+        - ./kafka-config/server.properties:/opt/kafka/config/server.properties
```

Comment:
If we aren't using volumes then `:z` might be needed here for selinux permissions 
```
volumes:
    - ./kafka-config/server.properties:/opt/kafka/config/server.properties:z
    - ./kafka-data:/var/lib/kafka:z
```